GameData/RealismOverhaul/RO_DependentMods/RO_AJE.cfg

```diff
@@ -37,6 +37,177 @@
 	%title = Avon RB-146 Mk.302 Turbojet
 }
 
+// Clone aje_j57 to be J48
++PART[aje_j75]:FOR[RealismOverhaul]
```

Comment:
Typo, looks like should be `aje_j57`

Reply:
Both aje_j57 and aje_j75 exist in AJE.  Which is intended?

Reply:
they're nearly identical. In this case, I meant to clone aje_j57, but apparently cloned aje_j75 instead, and configured the part accordingly, so it should stay as j75